skimage/filters/tests/test_gaussian.py

```diff
@@ -1,16 +1,19 @@
 import numpy as np
-from numpy.testing import assert_raises
+import pytest
 from skimage.filters._gaussian import gaussian
 from skimage._shared._warnings import expected_warnings
 
 
 def test_negative_sigma():
     a = np.zeros((3, 3))
     a[1, 1] = 1.
-    assert_raises(ValueError, gaussian, a, sigma=-1.0)
-    assert_raises(ValueError, gaussian, a, sigma=[-1.0, 1.0])
-    assert_raises(ValueError, gaussian, a,
-                  sigma=np.asarray([-1.0, 1.0]))
+    with pytest.raises(ValueError):
+        gaussian(a, sigma=-1.0)
+    with pytest.raises(ValueError):
+        gaussian(a, sigma=[-1.0, 1.0])
+    with pytest.raises(ValueError):
+        gaussian(a,
+                 sigma=np.asarray([-1.0, 1.0]))
```

Comment:
@jni Because `numpy.testing.assert_raises` use `nose` under the hood:
https://github.com/scikit-image/scikit-image/pull/2468#discussion_r97205194
https://github.com/numpy/numpy/blob/v1.12.0/numpy/testing/utils.py#L1179-L1206